src/applications/rated-disabilities/util/index.js

```diff
@@ -5,12 +5,8 @@ const SERVER_ERROR_REGEX = /^5\d{2}$/;
 const CLIENT_ERROR_REGEX = /^4\d{2}$/;
 
 export async function getData(apiRoute, options) {
-  try {
-    const response = await apiRequest(apiRoute, options);
-    return response.data.attributes;
-  } catch (error) {
-    return error;
-  }
+  const response = await apiRequest(apiRoute, options);
```

Comment:
This is the core fix and should make it so that the try-catch in the `useEffect` block in the `AppContent` component is able to tell if there is an error from the API